vnext/Microsoft.ReactNative/Fabric/FabricUIManagerModule.cpp

```diff
@@ -0,0 +1,768 @@
+    runtimeExecutor_([this, ownerBox = ownerBox_](facebook::jsi::Runtime &runtime) {
+      auto owner = ownerBox->owner.lock();
+      if (!owner) {
+        return;
+      }
+
+      this->beat(runtime);
+    });
+  }
+
+  void induce() const override {
+    tick();
+  }
+
+  void request() const override {
+    bool alreadyRequested = isRequested_;
+    EventBeat::request();
+    if (!alreadyRequested) {
+      if (auto uiManager = uiManager_.lock()) {
+        // Notifies that an event will be dispatched (e.g. LayoutEvent)
+        // uiManager->onRequestEventBeat();
+      }
+    }
+  }
+
+ private:
+  // EventBeatManager *eventBeatManager_;
+  xaml::Media::CompositionTarget::Rendering_revoker m_rendering;
+  winrt::Microsoft::ReactNative::ReactContext m_context;
+  facebook::react::RuntimeExecutor runtimeExecutor_;
+  std::weak_ptr<FabricUIManager> uiManager_;
+};
+
+std::shared_ptr<facebook::react::ComponentDescriptorProviderRegistry const> sharedProviderRegistry() {
+  static auto providerRegistry = []() -> std::shared_ptr<facebook::react::ComponentDescriptorProviderRegistry> {
+    auto providerRegistry = std::make_shared<facebook::react::ComponentDescriptorProviderRegistry>();
+    providerRegistry->add(
+        facebook::react::concreteComponentDescriptorProvider<facebook::react::ImageComponentDescriptor>());
+    providerRegistry->add(
+        facebook::react::concreteComponentDescriptorProvider<facebook::react::ParagraphComponentDescriptor>());
+    providerRegistry->add(
+        facebook::react::concreteComponentDescriptorProvider<facebook::react::RawTextComponentDescriptor>());
+    providerRegistry->add(
+        facebook::react::concreteComponentDescriptorProvider<facebook::react::ScrollViewComponentDescriptor>());
+    providerRegistry->add(
+        facebook::react::concreteComponentDescriptorProvider<facebook::react::TextComponentDescriptor>());
+    providerRegistry->add(
+        facebook::react::concreteComponentDescriptorProvider<facebook::react::TextInputComponentDescriptor>());
+    providerRegistry->add(
+        facebook::react::concreteComponentDescriptorProvider<facebook::react::ViewComponentDescriptor>());
+    return providerRegistry;
+  }();
+
+  return providerRegistry;
+}
+
+void FabricUIManager::installFabricUIManager( // in android this is Binding::installFabricUIManager(
+                                              // jni::alias_ref<JRuntimeExecutor::javaobject> runtimeExecutorHolder,
+                                              // jni::alias_ref<jobject> javaUIManager,
+                                              // EventBeatManager *eventBeatManager,
+                                              // jni::alias_ref<JavaMessageQueueThread::javaobject>
+                                              // jsMessageQueueThread, ComponentFactory *componentsRegistry,
+                                              // jni::alias_ref<jobject> reactNativeConfig
+    ) noexcept {
```

Comment:
does it matter what this is in android? consider removing if not or add a comment to explain why/if it has to be kept in sync or how